sky/global_user_state.py

```diff
@@ -426,11 +426,16 @@ def _get_cluster_launch_time(cluster_hash: str) -> Optional[Dict[str, Any]]:
 
 
 def _get_cluster_duration(cluster_hash: str) -> Optional[Dict[str, Any]]:
+    total_duration = 0
     usage_intervals = _get_cluster_usage_intervals(cluster_hash)
 
-    total_duration = 0
+    if usage_intervals is None:
+        return total_duration
+
     for i, (start_time, end_time) in enumerate(usage_intervals):
         # duration from latest start time to time of query
+        if start_time is None:
+            continue
```

Comment:
In what case will the `start_time` be `None`? Worth adding a comment for this.

Reply:
I guess there is no specific case where this would happen, so I can remove this logic